src/Campaigns/Blocks/CampaignList/block.json

```
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://json.schemastore.org/block.json",
+    "apiVersion": 2,
+    "name": "givewp/campaign-list-block",
+    "version": "1.0.0",
+    "title": "Campaign List Block",
```

We don't need the word block in our blocks as its already implied, also I actually think this should actually be called "Campaign Grid" now that I think of it.

```suggestion
    "title": "Campaign Grid",
```

---

Resolved: https://github.com/impress-org/givewp/pull/7708/commits/7706ddb39ae38e1ee023ad3d1d97b34b3586a8aa